resources/docs/TUTORIAL_1_BASICS.md

```diff
@@ -83,6 +84,18 @@ This should print:
 Sentence: "The grass is green ." - 5 Tokens
 ```
 
+You can write and provide your own wrapper around the tokenizer you want to use.  
```

Comment:
After the PR is merged, this text will be on the web page, but most users install flair through pip, i.e. they will not have access to this feature. Perhaps point out in the text that this is a feature only available on master branch currently.